### PR TITLE
shake on keys! :leaves:

### PR DESCRIPTION
**Pull request by NataliaLKB**
cc @tsop14

### CODE REVIEW

File: shake.css [hunk 1]
```diff
@@ -1,4 +1,7 @@
-.fc-item:hover {
+.shake-front-container {
+	display: inline-block !important;
+	position: relative !important;
+
```
Comment (NataliaLKB) [inline shake.css]: Whoops! Will remove.

File: script.js [hunk 2]
```diff
@@ -1,35 +1,47 @@
...
+
+            if (arraysAreEqual( typedKeys, keyCombo )) {
+                var containers = document.getElementsByClassName("fc-item__container");
+                var firstContainer = containers[0];
+                var i;
+
+                firstContainer.className += " shake-front-container";
+
+                for (i = containers.length - 1; i >= 0; i--) {
+                    containers[i].addEventListener("mouseover", shakeOnHover, false);
```
Comment (tsop14) [inline script.js:49]: does containers[0] need this added?

Comment (NataliaLKB) [inline script.js:49]: Doesn't hurt, but doesn't need it either.

Comment (NataliaLKB) [inline script.js:49]: feel free to change it in subsequent PRs :smile:

**Comment by tsop14** [comment]
:+1:

**Pull request merged**